malariagen_data/anoph/base_params.py

```diff
@@ -279,6 +281,8 @@ def validate_sample_selection_params(
     """,
 ]
 
+max_missing_an_default: max_missing_an = 0
+
```

Comment:
I'm fine to have these here, but it could be argued that these defaults are less "base" defaults but rather more specific defaults that only apply when you want to run certain types of population structure analyses such as PCA or NJT, where it matters to have segregating variants and you want to minimise missingness. As such they could possibly live better in the pca_params module.

Reply:
Thanks @alimanfoo . Originally, I put `max_missing_an_default` and `min_minor_ac_default` in `pca_params.py` before I realised that `min_minor_ac` and `max_missing_an` were already in `base_params.py`.
I assumed we shouldn't have duplicate params in different modules.

Reply:
I wonder if something like this works in `pca_params.py`:
```python
from . import base_params
min_minor_ac_default: base_params.min_minor_ac = 2
```
To allow:
```python
        min_minor_ac: Optional[
            base_params.min_minor_ac
        ] = pca_params.min_minor_ac_default,
```